source/pci/pci_test.go

```diff
@@ -232,7 +232,8 @@ func TestPciSource(t *testing.T) {
 	// Run test cases
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			hostpath.SysfsDir = hostpath.HostDir(filepath.Join(packagePath, "testdata", tc.rootfs, "sys"))
+			mockSysfsPath := filepath.Join(packagePath, "..", "..", "testdata", "source", "pci", tc.rootfs, "sys")
```

Comment:
Is the package path used for anything else other than the testdata dir? should we just set that directly instead?

Reply:
umm you maybe right, `packagePath` has no other use, so we could even trim the `init` func at the beginning of the file

Reply:
Done